server/cmwell-dc/src/main/scala/cmwell/dc/stream/SingleMachineInfotonIngester.scala

```diff
@@ -59,7 +59,13 @@ object SingleMachineInfotonIngester extends LazyLogging {
         case (infotonSeq, state) => {
           val payloadBuilder = new ByteStringBuilder
           // no need for end line because each line in already suffixed with it
-          infotonSeq.foreach(payloadBuilder ++= _.data)
+          infotonSeq.foreach{i=>
+            val firstLine = i.data.takeWhile(_ != stream.newLine)
+            if(i.data.takeWhile(_ != stream.newLine).contains(ByteString("meta/sys#uuid")))
+              payloadBuilder ++= i.data.take(firstLine.length)
```

Comment:
1. Third time the same `take`.
2. If the first line is uuid take *only* the first line?

Reply:
1.done. I changed it to drop
2. it's working correctly